### PR TITLE
Fixed mute button default state

### LINKED ISSUE #140: Muted icon not showing
A `<video muted>` should show the muted icon in the mute button on page load.

https://codepen.io/luwes/pen/yLPypdG?editors=1000

This worked in v0.2
https://codepen.io/luwes/pen/vYWEdgJ?editors=1000

### PR DESCRIPTION
**Pull request by heff**
fixes #140

Also updated media controller APIs to include needed functions and centralized logic.

### CODE REVIEW

File: src/js/media-controller.js [hunk 1]
```diff
@@ -503,36 +493,41 @@ class MediaController extends MediaContainer {
...
-        this.media.muted
+        this.muted
       );
       propagateMediaState(
         [el],
         MediaUIAttributes.MEDIA_VOLUME,
-        this.media.volume
+        this.volume
+      );
+      propagateMediaState(
```
Comment (heff) [inline src/js/media-controller.js:509]: This is the thing that specifically fixes the muted default.

File: snowpack.dev.config.cjs [hunk 2]
```diff
@@ -8,6 +8,7 @@ module.exports = {
     'src/js': { url: '/dist' },
     // Mount "public" to the root URL path ("/*") and serve files with zero transformations:
     examples: { url: '/examples', static: true, resolve: false },
+    sandbox: { url: '/sandbox', static: true, resolve: false },
```
Comment (cjpillsbury) [inline snowpack.dev.config.cjs:11]: ? was this intentional?

File: src/js/media-controller.js [hunk 3]
```diff
@@ -663,8 +694,9 @@ const getMediaUIAttributesFrom = (child) => {
   );
 };
 
-const isMediaStateReceiver = (child) =>
-  !!getMediaUIAttributesFrom(child).length;
+const isMediaStateReceiver = (child) => {
+  return !!getMediaUIAttributesFrom(child).length;
```
Comment (cjpillsbury) [inline src/js/media-controller.js:698]: needed a breakpoint? 😄

File: src/js/media-controller.js [hunk 4]
```diff
@@ -247,26 +247,15 @@ class MediaController extends MediaContainer {
...
-        } else if (volume < 0.75) {
-          level = 'medium';
-        }
-
-        this.propagateMediaState(MediaUIAttributes.MEDIA_MUTED, muted);
-        this.propagateMediaState(MediaUIAttributes.MEDIA_VOLUME, volume);
-        this.propagateMediaState(MediaUIAttributes.MEDIA_VOLUME_LEVEL, level);
+        this.propagateMediaState(MediaUIAttributes.MEDIA_MUTED, this.muted);
+        this.propagateMediaState(MediaUIAttributes.MEDIA_VOLUME, this.volume);
+        this.propagateMediaState(
```
Comment (cjpillsbury) [inline src/js/media-controller.js:255]: 👍

**Review by cjpillsbury — APPROVED** [review]
snowpack.dev.config.cjs commit looks sus, but pre-approving to avoid friction.

File: snowpack.dev.config.cjs (hunk 2 above)
Comment (heff) [reply snowpack.dev.config.cjs:11]: Yeah, intentional. Before switching to snowflake this just worked. I've always appreciated having a sandbox folder I can toss test files in and not worry about messing with actual project file like demos and examples. (i.e. [video.js/sandbox](https://github.com/videojs/video.js/tree/main/sandbox)) I should probably add a .example file too to make it clear what you can do with it.

File: src/js/media-controller.js (hunk 3 above)
Comment (heff) [reply src/js/media-controller.js:698]: Ah, didn't mean to leave this changed, but yes! Somewhere to drop a console.log.

**Pull request merged**